extension/framework/goplugin/runner/runner.go

```diff
@@ -180,9 +182,19 @@ func (testRunner *TestRunner) runSingle(t ginkgo.GinkgoTestingT, reporter *Repor
 	// create env
 	beforeStartHook := func(env *goplugin.Environment) error {
 		// db
-		dbFileName := dbBaseFileName + "_" + uuid.NewV4().String()
-		dbConnString := fmt.Sprintf("file:%s?mode=memory&cache=shared", dbFileName)
-		db, err := gohan_db.ConnectDB("sqlite3", dbConnString, gohan_db.DefaultMaxOpenConn, options.Default())
+		dbType := ""
+		dbConn := ""
+
+		if os.Getenv("MYSQL_TEST") == "true" {
+			dbType = "mysql"
+			dbConn = "root@/gohan_test"
```

Comment:
what about taking env variable MYSQL_DSN with this as default?

Reply:
This variable is never used in gohan. What kind of usage you have in mind?

Reply:
I mean that I have different credentials to my local mysql server. I would need to recompile gohan if I would like to run tests. Getting DSN string from env variable is much better solution.

Reply:
done, added optional DSN parameter